src/occupation.jl

```diff
@@ -86,27 +92,21 @@ function find_occupation_fermi_metal(basis, energies, Psi)
     εF = Roots.find_zero(εF -> compute_n_elec(εF) - n_electrons, (min_ε, max_ε),
                          Roots.Bisection())
     @assert compute_n_elec(εF) ≈ n_electrons
+    minocc = maximum(minimum.(compute_occupation(εF)))
+    if minocc > .01
+        @warn "One kpoint has minimum occupation $minocc, you should increase the number of bands"
```

Comment:
I'd specify the issue more explicitly. `"Very low occupation of $minocc detected on one kpoint, you should ..."`